src/test/java/com/wikia/webdriver/common/templates/NewTestTemplate.java

```diff
@@ -8,25 +14,22 @@
 import com.wikia.webdriver.common.driverprovider.NewDriverProvider;
 import com.wikia.webdriver.common.driverprovider.UseUnstablePageLoadStrategy;
 
-import org.testng.SkipException;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-
-import java.lang.reflect.Method;
-
 public class NewTestTemplate extends NewTestTemplateCore {
 
+  private void setTestProperty(String key, String value) {
+    if (!"".equals(value)) {
+      Configuration.setTestValue(key, value);
+    }
+  }
+
   @BeforeMethod(alwaysRun = true)
   public void start(Method method, Object[] data) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "start" is 13 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)